src/autoscaler/db/sqldb/instancemetrics_sqldb_test.go

```diff
@@ -279,12 +268,6 @@ var _ = Describe("InstancemetricsSqldb", func() {
 			err = idb.SaveMetric(metric)
 			Expect(err).NotTo(HaveOccurred())
 
-			metric.InstanceIndex = 0
-			metric.CollectedAt = 222222
-			metric.Value = "654321"
-			metric.Timestamp = 111100
-			err = idb.SaveMetric(metric)
-			Expect(err).NotTo(HaveOccurred())
 
```

Comment:
why delete this?

Reply:
If we set the primary key for the instancemetrics table (appid+instance_index+metric_type+timestamp), it will not allowed to store same records. This record is duplicate with the first one.